releases/release-1.13/release-notes-draft.md

```diff
@@ -422,4 +422,43 @@ SIG Windows focused on improving reliability for Windows and Kubernetes support
 - Handle Windows named pipes in host mounts. ([#69484](https://github.com/kubernetes/kubernetes/pull/69484), [@ddebroy](https://github.com/ddebroy))
 - Fixed inconsistency in windows kernel proxy when updating HNS policy. ([#68923](https://github.com/kubernetes/kubernetes/pull/68923), [@delulu](https://github.com/delulu))
 
-
+##  External Dependencies
+
+- Default etcd server was updated to v3.2.24. ([#68318](https://github.com/kubernetes/kubernetes/pull/68318))
+- Rescheduler is unchanged from v1.11: v0.4.0. ([#65454](https://github.com/kubernetes/kubernetes/pull/65454))
+- The list of validated docker versions was updated to 1.11.1, 1.12.1, 1.13.1, 17.03, 17.06, 17.09, 18.06. ([#68495](https://github.com/kubernetes/kubernetes/pull/68495))
+- The default Go version was updated to 1.10.4. ([68802](https://github.com/kubernetes/kubernetes/pull/68802))
+- The minimum supported Go version was updated to 1.10.2 ([#63412](https://github.com/kubernetes/kubernetes/pull/63412))
+- CNI is unchanged from v1.10: v0.6.0 ([#51250](https://github.com/kubernetes/kubernetes/pull/51250))
```

Comment:
[This is still 0.6.0](https://github.com/kubernetes/kubernetes/blob/ac217310bf984d433e89864fedb1a161270fcfc3/build/debian-hyperkube-base/Makefile#L28)

Reply:
I am finding this text confusing i.e., it doesnt clearly seem to indicate that 1.10 is k8s version, while and CNI remains at 0.6.0. Can we update this along the lines of

CNI is unchanged at v0.6.0, since kubernetes v1.10 release

Reply:
Looking at other entries below, this seems to be a standard convention. May be its just me its confusing. if changing consider updating all the similar lines below eg: kube-dns, influxdb etc